Getting_and_Cleaning_Data/Tidying_Data_with_tidyr/lesson.yaml

```diff
@@ -227,7 +227,7 @@
   Hint: Call bind_rows() with two arguments, passed and failed (in that order), to join the two tables.
 
 - Class: text
-  Output: Of course, we could arrange the rows however we wish at this point, but the important thing is that each row is an observation, each column is a variable, and the table contains a single observational unit. Thus, the data are tidy.
+  Output: Of course, we could arrange the rows as we wish at this point, but the important thing is that each row is an observation, each column is a variable, and the table contains a single observational unit. Thus, the data are tidy.
```

Comment:
I don't see how changing `however` to `as` clarifies this statement. I like your other changes.